esmf_utils/OutputInfo.F90

```diff
@@ -123,37 +130,47 @@ function get_vertical_dim_spec_names_bundle_info(info, rc) result(names)
       integer, optional, intent(out) :: rc
       integer :: status
       integer :: i
-      character(len=:), allocatable :: name
+      character(len=ESMF_MAXSTR) :: spec_name
 
+      spec_name = EMPTY_STRING
       names = StringVector()
       do i=1, size(info)
-         name = get_vertical_dim_spec_info(info(i), _RC)
-         if(find_index(names, name) == 0) call names%push_back(name)
+         spec_name = get_vertical_dim_spec_info(info(i), _RC)
+         _ASSERT(spec_name /= EMPTY_STRING, 'No vertical dim spec found.')
+         if(find_index(names, spec_name) == 0) call names%push_back(trim(spec_name))
       end do
       _RETURN(_SUCCESS)
 
    end function get_vertical_dim_spec_names_bundle_info
 
    function get_vertical_dim_spec_name_field(field, rc) result(spec_name)
-      character(len=:), allocatable :: spec_name
+      character(len=ESMF_MAXSTR) :: spec_name
       type(ESMF_Field), intent(in) :: field
       integer, optional, intent(out) :: rc
       integer :: status
       type(ESMF_Info) :: info
 
+      spec_name = EMPTY_STRING
```

Comment:
`EMPTY_STRING` seems a bit verbose for this.    We don't define ZERO to be 0, so why not just use `""` here (and elsewhere?   Don't change.  Just curious if there is a specific motivation here.

Reply:
It's an aesthetic choice. All these being equal, I like to make code read well.